scripts/e2e_test.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e
+
+AWM_RELAYER_PATH=$(
```

Comment:
Let's name this `RELAYER_PATH` to be consistent with the other scripts (those changes may not have all been merged)